packages/xrpl/test/integration/transactions/oracleSet.test.ts

```diff
@@ -39,6 +43,14 @@ describe('OracleSet', function () {
               Scale: 3,
             },
           },
+          {
+            PriceData: {
+              BaseAsset: 'XRP',
+              QuoteAsset: 'INR',
+              AssetPrice: BigInt(MAX_64_BIT_UNSIGNED_INT),
```

Comment:
Does this not work out of the box if you specify the number as a string?

Reply:
@mvadari  you are correct, thanks for pointing it out. I didn't realize that `str` type preserves the range of the UINT64 type.

@khancode I'd like to close this PR -- Modifying the parseFloat method might not be needed and I don't have the time to look into it.

However, this [if-condition](https://github.com/XRPLF/xrpl.js/blob/0a45c58934620d8cad127a8d5652e7bc9f377836/packages/xrpl/src/models/transactions/oracleSet.ts#L149) needs to be removed for this integration test to work. Presently, developers cannot use `str` type inputs for the `AssetPrice` field in `OracleSet` transaction.

Reply:
Then create another PR to remove the if-condition?

Reply:
I can't take it up for the next two weeks. I want to complete other tasks, I can take it up after that.

Reply:
Then follow the process of creating a ticket and moving it to the appropriate sprint to work on later.

Reply:
Let's discuss this offline but close this PR and create a ticket to track the fix.